uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
This is where I fixed the at-the-door reg form to just include ``regform.html`` instead of re-defining all the same field fields.